rest-client-base/src/main/java/com/wultra/core/rest/client/base/RestClientConfiguration.java

```diff
@@ -337,6 +340,22 @@ public void setUseCustomKeyStore(boolean useCustomKeyStore) {
         this.useCustomKeyStore = useCustomKeyStore;
     }
 
+    /**
+     * Get byte data with the key store.
+     * @return Byte data with key store data.
+     */
+    public ByteBuffer getKeyStoreBytes() {
+        return keyStoreBytes;
+    }
+
+    /**
+     * Set byte data with the key store.
+     * @param keyStoreBytes Byte data with the key store.
+     */
+    public void setKeyStoreBytes(ByteBuffer keyStoreBytes) {
+        this.keyStoreBytes = keyStoreBytes;
```

Comment:
I am not sure about storing a direct reference to a `ByteBuffer` object. I think would be better to either store its copy or use `byte[]`. The reason is that the `ByteBuffer` can be changed later in the code which calls this method and this impacts the stored data as a side effect.